modules/autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/viewing/AutofillManagementDisabledMode.kt

```diff
@@ -66,15 +66,18 @@ class AutofillManagementDisabledMode : Fragment() {
 
     @SuppressLint("InlinedApi")
     private fun launchDeviceAuthEnrollment() {
-        if (appBuildConfig.sdkInt >= Build.VERSION_CODES.R) {
-            requireActivity().startActivity(Intent(android.provider.Settings.ACTION_BIOMETRIC_ENROLL))
-        } else {
-            if (appBuildConfig.sdkInt >= Build.VERSION_CODES.P) {
+        when {
+            appBuildConfig.sdkInt >= Build.VERSION_CODES.R ->
+                requireActivity().startActivity(Intent(android.provider.Settings.ACTION_BIOMETRIC_ENROLL))
+            appBuildConfig.manufacturer == "Xiaomi" ->
```

Comment:
I don't know what will happen on a Xiaomi device newer than Android R. Is it worth doing the manufacturer check first so that all Xiaomi devices get the generic settings, regardless of OS version?

Reply:
Good point. Updated it for the sake of safety. I don't think it's a bad info anyway.